src/js/engagement_view/src/modules/uploads/plugins/apiRequests.tsx

```diff
@@ -50,6 +50,6 @@ export const uploadFilesToDgraph = async (payload: PluginPayload ): Promise<bool
         }
     );
     const body = await res.json();
-    console.log("body", body)
+    // console.log("body", body)
```

Comment:
hmm

Reply:
I commented this out because I don't think users should ever be in the console, however I wanted to keep that there in case I want to troubleshoot it in the future, I know it's really minor, it's just easier that way, but I can take it out if you'd prefer

Reply:
Yeah, maybe we should console.debug here.